src/main/java/eu/bbmri_eric/negotiator/governance/resource/dto/ResourceDTO.java

```diff
@@ -19,12 +19,15 @@
 @AllArgsConstructor
 @ToString
 @JsonInclude(value = JsonInclude.Include.NON_NULL)
-@Deprecated
 public class ResourceDTO {
```

Comment:
please use the ResourceResponseModel and leave this deprecated

Reply:
Yes I already use it in the post and put controllers but there are some controllers (not related to this PR) where it is still used. Additional refactoring should be done here. Also, if we decide to uniform things, the same is valid for the assemblers and other output DTOs: shall also the others will be transformed as ResourceResponseModel is ?